REQUIRE

```diff
@@ -3,3 +3,4 @@ julia 0.4
 Reexport
 Plots
 LearnBase
+ValueHistories
```

Comment:
I'd really like to avoid adding dependencies, unless absolutely necessary.  There are a few options... one of them is simply wrapping the include in a try block, so that if the user doesn't have the package it just doesn't include this functionality.  Then we don't have to add it to the  REQUIRE file.